src/explore-education-statistics-admin/src/pages/release/data/utils/__tests__/getApiDataSetFilterMappings.test.ts

```diff
@@ -0,0 +1,365 @@
+          },
+          sourceKey: 'Filter3Option4Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option5Key',
+          label: 'Filter 3 Option 5',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option5Key',
+          publicId: 'filter-3-option-5-public-id',
+          source: {
+            label: 'Filter 3 Option 5',
+          },
+          sourceKey: 'Filter3Option5Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option6Key',
+          label: 'Filter 3 Option 6',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option6Key',
+          publicId: 'filter-3-option-6-public-id',
+          source: {
+            label: 'Filter 3 Option 6',
+          },
+          sourceKey: 'Filter3Option6Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option7Key',
+          label: 'Filter 3 Option 7',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option7Key',
+          publicId: 'filter-3-option-7-public-id',
+          source: {
+            label: 'Filter 3 Option 7',
+          },
+          sourceKey: 'Filter3Option7Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option8Key',
+          label: 'Filter 3 Option 8',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option8Key',
+          publicId: 'filter-3-option-8-public-id',
+          source: {
+            label: 'Filter 3 Option 8',
+          },
+          sourceKey: 'Filter3Option8Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option9Key',
+          label: 'Filter 3 Option 9',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option9Key',
+          publicId: 'filter-3-option-9-public-id',
+          source: {
+            label: 'Filter 3 Option 9',
+          },
+          sourceKey: 'Filter3Option9Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option10Key',
+          label: 'Filter 3 Option 10',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option10Key',
+          publicId: 'filter-3-option-10-public-id',
+          source: {
+            label: 'Filter 3 Option 10',
+          },
+          sourceKey: 'Filter3Option10Key',
+          type: 'AutoMapped',
+        },
+      },
+      {
+        candidate: {
+          key: 'Filter3Option11Key',
+          label: 'Filter 3 Option 11',
+        },
+        mapping: {
+          candidateKey: 'Filter3Option11Key',
+          publicId: 'filter-3-option-11-public-id',
+          source: {
+            label: 'Filter 3 Option 11',
+          },
+          sourceKey: 'Filter3Option11Key',
+          type: 'AutoMapped',
+        },
+      },
+    ];
```

Comment:
Know this is using shared test data, but testing 11 options seems a bit excessive. Perhaps we could trim just a few items?

Reply:
Yeah I've cut it down here and for locations, it was originally to check that only 10 were rendered in the auto mapped table but now that's tested separately so it's not needed.